### ISSUE TITLE
LazyRandomVector not defined

### ISSUE THREAD

**Issue by cscherrer** [zenna/Sigma.jl#49]
Following `Pkg.clone("https://github.com/zenna/Sigma.jl.git")`, I tried `using Sigma`, and go this:

```
julia> using Sigma
ERROR: LazyRandomVector not defined
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in reload_path at loading.jl:152
 in _require at loading.jl:67
 in require at loading.jl:51
while loading /home/chad/.julia/v0.3/Sigma/src/omega.jl, in expression starting on line 25
while loading /home/chad/.julia/v0.3/Sigma/src/Sigma.jl, in expression starting on line 160
```

Is there a package that defines this, that should be listed as a dependency? Thanks.

**Comment by zenna** [comment]
Yes this comes from AbstractDomains.jl, which is automatically installed with Sigma but the installed version is behind the latest version.  To fix it you could do Pkg.checkout("AbstractDomains") but there are bigger issues at the moment.  Give me a little time and I'll update you when it's in working order (actually this is why I decided not to speak at the end of PPAML, because I didn't want everyone trying to download it and failing)

**Comment by cscherrer** [comment]
Sounds good, I'll wait for your go-ahead to give it another shot. Thanks.
